<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QFrame" name="myframe">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>951</width>
     <height>501</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="group3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>631</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>亮度调节</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lightlevelsub">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QSlider" name="lightlevelslider">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>341</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lightlevel">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lightleveladd">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>色温调节</string>
     </property>
    </widget>
    <widget class="QPushButton" name="colorlevelsub">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QSlider" name="colorlevelslider">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>341</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="colorlevel">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="text">
      <string>30</string>
     </property>
    </widget>
    <widget class="QPushButton" name="colorleveladd">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>亮度</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lightlevelset">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>色温</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="colorlevelset">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>保存自定义</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="temp">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>℃</string>
     </property>
    </widget>
    <widget class="QLabel" name="hum">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="text">
      <string>RH</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>湿度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>实时时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="time">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2017/9/11 12:15:31</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>串口号</string>
     </property>
    </widget>
    <widget class="QComboBox" name="PortBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>智能LED台灯</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="group2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="custommode">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>自定义模式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="aimode">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>智能模式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="sleepmode">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>睡眠模式</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>151</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>当前坐姿距离</string>
     </property>
    </widget>
    <widget class="QLabel" name="distance">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>cm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bestdistanceset">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>设置最佳坐姿</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>最佳坐姿距离</string>
     </property>
    </widget>
    <widget class="QLabel" name="bestdistance">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>cm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>近视预警</string>
     </property>
    </widget>
    <widget class="QLabel" name="alert">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="mainswitch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
